### PR TITLE
Supporting CORS-RFC1918

### PR DESCRIPTION
**Pull request by e-bits**
# Description

Chrome has implemented CORS-RFC1918, which prevents public network resources from requesting private-network resources - unless the public-network resource is secure (HTTPS) and the private-network resource provides appropriate (yet-undefined) CORS headers.

There's also a Chrome flag you can change to disable the new behavior for now:
`chrome://flags/#block-insecure-private-network-requests`

Disabling that flag does mean you're re-opening the security hole that Chrome's new behavior is meant to close.

Source: https://stackoverflow.com/questions/66534759/chrome-cors-error-on-request-to-localhost-dev-server-from-remote-site

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] tested with Brave 1.26.67

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have updated changelog.md with changes made

### CODE REVIEW

**Review by davidmckenzie — DISMISSED** [review]
Looks good thanks mate!

File: server/plugins/Telegram.js [hunk 1]
```diff
@@ -15,12 +15,14 @@ function run(trigger, scope, data, config, callback) {
         } else {
             //Notification formatted in Markdown for pretty notifications
             var notificationText = `*${data.agency} - ${data.alias}*\n` + 
-                                    `Message: ${data.message}`;
+                                    `${data.message}\n` + 
+                                    `[Google Maps](https://maps.google.com/maps?q=${data.message})`;
```
Comment (DanrwAU) [inline server/plugins/Telegram.js:19]: Why is this here? Looks like a custom change that shouldn't be in this PR

File: server/plugins/Telegram.js [hunk 2]
```diff
@@ -15,12 +15,14 @@ function run(trigger, scope, data, config, callback) {
...
             
             t.sendMessage({
                 chat_id: tConf.chat,
                 text: notificationText,
-                parse_mode: "Markdown"
+                parse_mode: "Markdown",
+                disable_web_page_preview: "true"
```
Comment (DanrwAU) [inline server/plugins/Telegram.js:25]: Again - this doesn't seem to have anything to do with the original PR

**Review by DanrwAU — CHANGES_REQUESTED** [review]
See Comments. 

Some unrelated changes seem to have snuck into this

**Comment by marshyonline** [comment]
This will be good after requested changes are made

File: server/plugins/Telegram.js (hunk 1 above)
Comment (SloCompTech) [reply server/plugins/Telegram.js:19]: I agree

File: server/plugins/Telegram.js (hunk 2 above)
Comment (SloCompTech) [reply server/plugins/Telegram.js:25]: I agree

**Comment by e-bits** [comment]
> This will be good after requested changes are made

Sorry for this. I made a small personal change and forgot to change the branch. I cleaned up my mistake. Hopefully this is correct now.

Cheers e-bits
